src/testing/CalculatorDisplay.test.js

```diff
@@ -0,0 +1,30 @@
+import '@testing-library/jest-dom';
+import { render, fireEvent } from '@testing-library/react';
+import CalculatorDisplay from '../components/CalculatorDisplay';
+
+describe('Calculator', () => {
+  it('Should render correctly', () => {
+    const { getByText } = render(<CalculatorDisplay />);
+    expect(getByText('AC')).toBeInTheDocument();
+    expect(getByText('+/-')).toBeInTheDocument();
+    expect(getByText('%')).toBeInTheDocument();
+    expect(getByText('÷')).toBeInTheDocument();
+    expect(getByText('x')).toBeInTheDocument();
+    expect(getByText('-')).toBeInTheDocument();
+    expect(getByText('+')).toBeInTheDocument();
+    expect(getByText('.')).toBeInTheDocument();
+    expect(getByText('=')).toBeInTheDocument();
+  });
+});
+
+test('should show "0" by default', () => {
+  const { getByText } = render(<CalculatorDisplay />);
+  expect(getByText('0')).toBeInTheDocument();
```

Comment:
- [ ] One of your tests is failing. Kindly rectify it.

  ![image](https://user-images.githubusercontent.com/81468675/236203055-f4e6e459-a06b-43d9-b52a-68e9bd032d46.png)